### PR TITLE
update Android SDK to API level 33

### PR DESCRIPTION
**Pull request by thatfiredev**
Althought Android 13 is still in Beta 4, their [timeline](https://developer.android.com/about/versions/13/overview#timeline) indicates that it's already reached platform stability and recommends that apps ands libraries should start releasing compatible versions.

I think it's about time to have our libraries and sample apps updated, before the final release.

### CODE REVIEW

**Comment by thatfiredev** [comment]
Android 13 has been released: https://twitter.com/Android/status/1559235471770894338?t=USg5HCQ8ynDA2Z_ccI_NVQ&s=19

Guess I jinxed it. 😅

File: fix_android_dependencies.py
Comment (Dinesh154160) [inline fix_android_dependencies.py:1]: Ok

**Review by Dinesh154160 — COMMENTED** [review]
Ok

**Comment by Dinesh154160** [comment]
> Althought Android 13 is still in Beta 4, their [timeline](https://developer.android.com/about/versions/13/overview#timeline) indicates that it's already reached platform stability and recommends that apps ands libraries should start releasing compatible versions.
> 
> I think it's about time to have our libraries and sample apps updated, before the final release.

**Pull request merged**